api/default_networks.go

```diff
@@ -20,17 +19,29 @@ const (
 	sttSymbol                     = "STT"
 )
 
-var ganacheTokenAddress = common.HexToAddress("0x8571Ddc46b10d31EF963aF49b6C7799Ea7eff818")
+func proxyUrl(stageName, provider, chainName, networkName string) string {
+	return fmt.Sprintf("https://%s.api.status.im/%s/%s/%s/", stageName, provider, chainName, networkName)
+}
 
 func mainnet(stageName string) params.Network {
+	chainID := MainnetChainID
+	chainName := "ethereum"
+	networkName := "mainnet"
+
+	rpcProviders := []params.RpcProvider{
+		// Proxy providers
+		*params.NewProxyProvider(chainID, "proxy-nodefleet", proxyUrl(stageName, "nodefleet", chainName, networkName), false),
```

Comment:
Looks like all these strings (`"proxy-nodefleet"`, `"nodefleet"`, etc) should be defined as const variables.